xiblint/rules/named_colors.py

```diff
@@ -8,11 +8,15 @@ class NamedColors(Rule):
 
     Example configuration:
     {
-      "ignore_alpha": true
+      "ignore_alpha": true,
+      "allowed_colors": ["CustomRed", "CustomGreen"],
+      "allow_system_colors": true
```

Comment:
NIT: Lets present the options in order of importance
```suggestion
      "allowed_colors": ["CustomRed", "CustomGreen"],
      "allow_system_colors": true.
      "ignore_alpha": true
```